src/ProgramBlockEditor.js

```diff
@@ -1,22 +1,25 @@
 // @flow
 
-import { Button, Col, Row } from 'react-bootstrap';
-import {injectIntl} from 'react-intl';
+import { Button, Col, Container, Row } from 'react-bootstrap';
+import { injectIntl, FormattedMessage } from 'react-intl';
 import * as ProgramUtils from './ProgramUtils';
 import type {Program, SelectedAction} from './types';
 import React from 'react';
 import { ReactComponent as ArrowTurnLeft } from './svg/ArrowTurnLeft.svg';
 import { ReactComponent as ArrowTurnRight } from './svg/ArrowTurnRight.svg';
 import { ReactComponent as ArrowForward } from './svg/ArrowForward.svg';
-import { ReactComponent as AddIcon } from 'material-design-icons/content/svg/production/ic_add_24px.svg';
-import { ReactComponent as DeleteIcon } from 'material-design-icons/content/svg/production/ic_clear_24px.svg';
+import { ReactComponent as AddIcon } from './svg/Add.svg';
+import { ReactComponent as DeleteIcon } from './svg/Delete.svg';
+import { ReactComponent as PlayIcon } from './svg/Play.svg';
 import './ProgramBlockEditor.css';
 
 type ProgramBlockEditorProps = {
     intl: any,
     minVisibleSteps: number,
     program: Program,
     selectedAction: SelectedAction,
+    runButtonDisabled: string,
```

Comment:
Let's make `runButtonDisabled` into a `boolean `and implement the logic to determine if it's disabled in App.js

    runButtonDisabled={this.state.dashConnectionStatus !== 'connected'}